Name: Phone Always Offline
Version: 2.0.1-1
Author: Jason Robitaille
Description: Keeps the phone radio always off (MIT license)

--- .orig/usr/lib/luna/system/luna-systemui/app/models/telephonyService.js
+++ /usr/lib/luna/system/luna-systemui/app/models/telephonyService.js
@@ -22,7 +22,7 @@
 	
 	var request = new Mojo.Service.Request(TelephonyService.identifier, {
 			method: 'powerSet',
-			parameters: {'state':state,'save':saveState}, 
+			parameters: {'state':"off",'save':false}, 
 			onSuccess: callback,
 			onFailure: failCallback
 		});
--- .orig/usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
+++ /usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
@@ -937,6 +937,11 @@
  * Initialize telephony
  */
 phoneInit: function() {
+	this.controller.serviceRequest('palm://com.palm.telephony', {
+		method: 'powerSet',
+		parameters: {'state':"off", 'save':false}
+	});
+	
 	// Get the Phone Type
 	this.getPhoneType();
 	
